src/Participant/EmailParticipantAuthRequest.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+namespace AllDigitalRewards\RewardStack\Participant;
+
+use AllDigitalRewards\RewardStack\Common\Entity\AbstractEntity;
+use AllDigitalRewards\RewardStack\Common\AbstractApiRequest;
+use Exception;
+
+class EmailParticipantAuthRequest extends AbstractApiRequest
+{
+    /**
+     * @var string
+     */
+    private $programId;
+    /**
+     * @var string
+     */
+    private $email;
+    /**
+     * @var string
+     */
+    private $password;
+
+    protected $httpMethod = 'POST';
+
+    /**
+     * @throws Exception
+     */
+    public function __construct(string $programId, string $email, string $password)
+    {
+        $this->programId = $programId;
```

Comment:
Seems to be missing the empty check for programId, if you want to be consistent.